files/fr/mdn/writing_guidelines/howto/document_a_css_property/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Comment documenter une propriété CSS
+slug: MDN/Writing_guidelines/Howto/Document_a_CSS_property
+---
```

Comment:
Il faudrait ajouter la clé de métadonnées `l10n.sourceCommit` qui fait référence au commit de `mdn/content` au moment où la création a été réalisée.
Cette information permet notamment de suivre sur le long cours les évolutions nécessaires en répondant à la question "quel écart dois-je reporter en français depuis la dernière mise à jour de la page"

Pour l'ajouter, privilégier la forme 

```
l10n:
  sourceCommit: hashlong01920934829824dddddd
```

Reply:
C'est noté, je modifie !